dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/contig/ContigMapping.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.contig;
+
+import java.util.Map;
+
+public class ContigMapping {
+    private Map<String, String> contigMap;
+
+    public ContigMapping(Map<String, String> contigMap) {
+        this.contigMap = contigMap;
+    }
+
+    public ContigMapping(String mappingUrl) throws Exception {
+        this(new AssemblyReportParser(mappingUrl).getContigMap());
+    }
+
+    public String getGenbank(String refseqContig) {
+        if (contigMap.containsKey(refseqContig)) {
+            return contigMap.get(refseqContig);
+        } else {
+            throw new IllegalArgumentException("There's no exact mapping available for the contig " + refseqContig);
```

Comment:
although this class can not decide what to do in case that the refseq doesn't have equivalent, I'm still not convinced it's worth to throw here. Only one place uses this function, and that place has to ignore this error. moreover, I can expect that this will happen several times in a regular execution.

an alternative would be to provide another method with a default value:
```
    public String getGenbank(String refseqContig, String defaultContig) {
        if (contigMap.containsKey(refseqContig)) {
            return contigMap.get(refseqContig);
        } else {
            return defaultContig;
        }
    }
```